app/controllers/users_controller.rb

```diff
@@ -44,12 +57,23 @@ def add_contact
     redirect_to root_path
   end
 
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def search
-    @users = User.search(params[:search]).where.not(id: current_user.id)
-    @users_to_add = @users.reject do |user|
-      current_user.sent_contact_request?(user)
+    use_wildcards = true
+    @users = Set[]
+    @contacts = Set[]
+    if current_user && params[:search]
+      @user = User.find(current_user.id)
```

Comment:
I think that variable could be named `@current_user` to be more precise

Reply:
Same as above.